pdf.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Fpdf\Fpdf;
+
+require('vendor/autoload.php');
+
+/*
+ * Generates pdf using user input data.
+ *
+ * @param string $firstName
+ *   User's first name.
+ * @param string $lastName
+ *   User's last name.
+ * @param string $fullName
+ *   User's full name.
+ * @param string $image
+ *   User's image.
+ * @param string $email
+ *   User's email.
+ * @param string $number
+ *   User's contact number.
+ * @param string $marks
+ *   User's subject marks.
+ *
+ * @return void
+ */
+function getPdf(string $firstName, string $lastName, string $fullName, mixed $image, string $email, string $phone, string $markString): void {
+  $pdf = new Fpdf();
+  $pdf->AddPage();
+  $pdf->SetFont("Arial", "", 16);
+  // Prints personal Details.
+  $pdf->Cell(0, 10, "Personal details", 0, 1, 'C');
+  $pdf->Cell(50, 10, "First Name", 1, 0);
+  $pdf->Cell(0, 10, $firstName, 1, 1);
+  $pdf->Cell(50, 10, "Last Name", 1, 0);
+  $pdf->Cell(0, 10, $lastName, 1, 1);
+  $pdf->Cell(50, 10, "Full Name", 1, 0);
+  $pdf->Cell(0, 10, $fullName, 1, 1);
+  $pdf->Cell(50, 10, "E-mail", 1, 0);
+  $pdf->Cell(0, 10, $email, 1, 1);
+  $pdf->Cell(50, 10, "Phone", 1, 0);
+  $pdf->Cell(0, 10, $phone, 1, 1);
+  $pdf->Cell(0, 10, "", 0, 1);
+  // Prints subject name along with it's marks.
+  $pdf->Cell(0, 10, "Academic Details", 0, 1, 'C');
+  $pdf->Cell(100, 10, "Subject", 1, 0);
+  $pdf->Cell(0, 10, "Marks", 1, 1);
+  $marks = explode("\n", $markString);
+  foreach ($marks as $mark) {
+    list($subject, $score) = explode("|", $mark);
```

Comment:
Our requirement was that user have to enter data only in the this format ;
Subject | Marks
We need to validate that , currently we are allowing any kind of data to be submitted, for example if user enters 40 | English, or maybe some special characters like ``$%# | $#! ` we are still printing it which isn't right. Please update accordinly with proper validations

Reply:
OK. I should use js or PHP?

Reply:
First let's do it with PHP.
Then later we can add also use JS.

Reply:
PHP done.